src/components/HouseholdCloseVisit/index.js

```diff
@@ -13,28 +15,41 @@ import NavigationButtons from '../NavigationButtons';
 class HouseholdCloseVisit extends Component {
     static propTypes = {
         requestCloseHouseholdVisit: PropTypes.func.isRequired,
+        requestHouseholdVisits: PropTypes.func.isRequired,
         onPrevious: PropTypes.func.isRequired,
         onSubmit: PropTypes.func.isRequired,
         match: matchParamsIdPropTypes.isRequired,
         chapter: chapterPropTypes.isRequired,
         household: PropTypes.instanceOf(Household).isRequired,
+        householdVisits: PropTypes.arrayOf(
+            PropTypes.shape({})
+        ),
         saving: PropTypes.bool
     };
 
     static defaultProps = {
-        saving: false
+        saving: false,
+        householdVisits: []
     };
 
     constructor(props) {
         super(props);
-        this.state = {
-            visitDetails: {}
-        };
+        this.state = {};
+    }
+
+    componentWillMount() {
+        const {id, dwellingOrder, householdOrder} = this.props.match.params;
+        this.props.requestHouseholdVisits(id, dwellingOrder, householdOrder);
     }
 
     componentWillReceiveProps(nextProps) {
-        if (this.props.saving && !nextProps.saving) {
-            this.props.onSubmit(nextProps.household);
+        const {saving, household, householdVisits} = nextProps;
+        if (!isEmpty(householdVisits)) {
+            this.state.visitDetails = last(householdVisits);
+            this.state.visitDetails.start = moment(this.state.visitDetails.start).format('YYYY-MM-DD HH:mm');
```

Comment:
xq en ese formato?

Reply:
Este formato es actualmente el que utiliza el DatePicker por default, si queres otro lo cambio el formato en el pull del form-builder => https://github.com/indec-it/react-native-form-builder/pull/64

Reply:
No podes guardar como dato una fecha en formato A solo xq así lo acepta el componente del datepicker, en la DB tenes que guardar un `Date` tal cual es, es un problema del datepicker si lo necesita es formato latino o ISO.